### ISSUE TITLE
Bug report: random crop

### ISSUE THREAD

**Issue by ThisIsIsaac** [backtime92/CRAFT-Reimplementation#40]
If the thread goes into [this](https://github.com/backtime92/CRAFT-Reimplementation/blob/master/data_loader.py#L93) branch, incorrect images like below occur:

![ballet_106_102_after_processing](https://user-images.githubusercontent.com/49523965/68745000-23b21f80-0639-11ea-8336-d13d7b7d71be.jpg)

If it goes to the `else `branch, the results are fine:

![ballet_106_102_after_processing](https://user-images.githubusercontent.com/49523965/68745008-27de3d00-0639-11ea-8abf-897db563c27d.jpg)

I think this is because the coordinates of character boxes are not adjusted properly in the first branch.

**Comment by backtime92** [comment]
@ThisIsIsaac I had not noticed this bug, I will check it .Thanks for you comment.

**Comment by chenhaolin1989** [comment]
have u solved the bug? can u share the correct code? THX

**Comment by ThisIsIsaac** [comment]
@chenhaolin1989 
I quick-fixed it by getting rid of the branch. This isn't ideal as it won't apply certain distortions.

**Comment by chenhaolin1989** [comment]
Thanks, I try to just ignore the such crop augmentation and will check the performance change after I get the results.

**Comment by backtime92** [comment]
@ThisIsIsaac @chenhaolin1989 sorry for the late reply, I have checkit ,but i did not find the bug. Can you share me more details.

**Comment by chenhaolin1989** [comment]
@backtime92 I don't think it is a bug, while the existing random crop strategy may cause that many input samples don't have GT bbox which may decrease the performance.

**Comment by ThisIsIsaac** [comment]
@chenhaolin1989 Maybe we are seeing different outputs. In my case, it wasn't that they weren't boxed. There were boxes, but all the boxes were shifted to the right by a few centimeteres.
